app/views/layouts/_navbar.html.erb

```diff
@@ -27,10 +26,22 @@
                         <%= link_to('Recipes', recipes_path, class: "nav-link#{' active' if controller_name == 'recipes'}", aria_current: controller_name == 'recipes' ? 'page' : false) %>
                     </li>
 
-                    <%# TODO: Add link %>
+                    <% if logged_in? %>
+                    <li class="nav-item">
+                        <%= link_to('Liked Recipes', librarys_path, class: "nav-link#{' active' if controller_name == 'librarys'}", aria_current: controller_name == 'librarys' ? 'page' : false) %>
+                    </li>
+                    <li class="nav-item">
+                        <%= link_to('Inventory', inventory_items_path, class: "nav-link#{' active' if controller_name == 'ingredient_items'}", aria_current: controller_name == 'ingredient_items' ? 'page' : false) %>
+                    </li>
                     <li class="nav-item">
                         <%= link_to('My Profile', '#', class: "nav-link#{' active' if controller_name == 'users'}", aria_current: controller_name == 'users' ? 'page' : false) %>
                     </li>
+                    
+                    <% else %>
+                     <%= link_to('Login', login_path, class: "nav-link#{' active' if controller_name == 'users'}", aria_current: controller_name == 'users' ? 'page' : false) %>
```

Comment:
```suggestion
                     <%= link_to('Login', login_path, class: "nav-link#{' active' if current_page?(login_path) || current_page?(signup_path)}", aria_current: current_page?(login_path) || current_page?(signup_path) ? 'page' : false) %>
```
We want to make sure we're only adding the `active` class on the associated pages.